tests/test_bridgepoint/utils.py

```diff
@@ -375,8 +375,27 @@ def setUp(self):
         pe_pe2 = self.metamodel.new('PE_PE')
         o_obj = self.metamodel.new('O_OBJ')
         o_obj.Key_Lett = 'OBJECT'
+        o_obj.Name = 'OBJECT'
         xtuml.relate(o_obj, pe_pe2, 8001)
 
+        pe_pe3 = self.metamodel.new('PE_PE')
```

Comment:
Is this what the bridgepoint editor refers to as a published reference?

Reply:
Yes. The BridgePoint editor requires references to be published to use set operations for type checking reasons. This change brings the pyxtuml pre-builder into consistency with the BridgePoint parser.